sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```diff
@@ -137,6 +138,8 @@ instances:
     ##              Columns without a name are ignored. To skip a column, enter:
     ##                - {}
     ## 3. tags (optional) - A list of tags to apply to each metric.
+    ## 4. collection_interval (optional) - The frequency at which to collect the metrics.
+    ##                                    The default is the integration `min_collection_interval`.
```

Comment:
I would also add in this doc that it cannot be set lower than the `min_collection_interval`

Reply:
make sense. I will update the template